script.js

```diff
@@ -47,3 +47,661 @@ navLink3.addEventListener('click', () => {
+  featuredImage: 'src="./images/project1-alt.jpg" alt="project3-snapshot"',
+  technologies: ['html', 'css', 'javascript', 'github', 'Ruby', 'Bootstrap'],
+  liveLink: 'https://aquinasmoriasi.github.io/Personal-portfolio/',
+  sourceLink: 'https://github.com/Aquinasmoriasi/Personal-portfolio',
+};
+const projectTwoCardDt = {
+  name: 'Uber Navigation',
+  description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. ummy text of the printing and typesetting release of Lorem Ipsum has been the industry\'s  standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea',
+  featuredImage: 'src="./images/project2.jpg" alt="project4-snapshot"',
+  technologies: ['html', 'css', 'javascript', 'github', 'Ruby', 'Bootstrap'],
+  liveLink: 'https://aquinasmoriasi.github.io/Personal-portfolio/',
+  sourceLink: 'https://github.com/Aquinasmoriasi/Personal-portfolio',
+};
+
+const pj1 = `
+<h2 id="pj1-h2">${projectOneCard.name}</h2>
+  <span class="close-btn">&times;</span>
+<div class="highlights">
+    <span class="canopy4">CANOPY</span>
+       <span>
+       <svg width="8" height="8" viewBox="0 0 8 8" fill="none" xmlns="http://www.w3.org/2000/svg">
+       <circle cx="4" cy="4" r="4" fill="#C1C7D0"/>
+       </svg>
+       </span>
+       <span class="not-canopy4">Back End Dev</span>
+       <span class="not-canopy-alt">Full Stack Dev</span>
+       <span>
+       <svg width="8" height="8" viewBox="0 0 8 8" fill="none" xmlns="http://www.w3.org/2000/svg">
+       <circle cx="4" cy="4" r="4" fill="#C1C7D0"/>
+       </svg>
+       </span>
+       <span class="not-canopy4">2015</span>
+    </div>
```

Comment:
[Optional]
- [ ] Instead of creating each card individually, you can create a function that accepts the content of the card to create a card and pass related data to that function. This way you can keep your code cleaner and simpler.

Reply:
Thank you, I don't like messy code too, I just had a bit if time  constraint. I will effect those changes.